pom.xml

```diff
@@ -1342,19 +1342,15 @@
         <version>3.1</version>
       </dependency>
       <!--
-        In the server, we can use a newer version than Fedora has.
-        This one also has a source jar!
+        Fedora now has com.io7m.xom:xom 1.2.10,
+        so just use the one that defined in parent
```

Comment:
@definite you say that, but then you specify the version a few lines down anyway. We should be making server less dependent on zanata-parent anyway, so I suggest removing this comment.